compiler/rustc_feature/src/active.rs

```diff
@@ -533,6 +533,8 @@ declare_features! (
     (active, type_changing_struct_update, "1.58.0", Some(86555), None),
     /// Enables rustc to generate code that instructs libstd to NOT ignore SIGPIPE.
     (active, unix_sigpipe, "1.65.0", Some(97889), None),
+    /// Allows a safe conversion from `&mut T` to `&mut UnsafeCell<T>`.
+    (incomplete, unsafe_cell_from_mut, "1.70.0", Some(111645), None),
```

Comment:
This isn't necessary for library features, the feature name will picked up from the `#[unstable]` attribute directly.

Reply:
I see, thank you.